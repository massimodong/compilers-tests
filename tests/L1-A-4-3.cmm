struct float P ;
int DWYmSt8iQjbib3 ;
int [ 954559328 ] = ! ! WiK_2zhBYs7G = I6C0MI5 ;
int OgYuDV_sJynHVGE = ! CPgg = AzGBC , , VHqSkBd ;
int = jBbI9wnrQlvJ1ztU ;
float IQgU4MTZE4JQvUu = ;
}_iZlacoWpSgU ( ) {
}
