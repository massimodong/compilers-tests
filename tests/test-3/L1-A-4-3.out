Error Type B at Line 1
Error Type B at Line 3
Error Type B at Line 4
Error Type B at Line 5
Error Type B at Line 6
